src/pages/admin/AdminDashboard.jsx

```diff
@@ -17,6 +17,10 @@ export const AdminDashboard = () => {
     <Container>
       <SideNav />
       <Body />
+      <AllOrders/>

```

Comment:
@Sodeeqishola5  can you move all this to the body component?

Reply:
Ok I will